mne/decoding/receptive_field.py

```diff
@@ -294,7 +324,33 @@ def _check_dimensions(self, X, y, predict=False):
         return X, y
 
 
-def _delay_time_series(X, tmin, tmax, sfreq, newaxis=0, axis=0):
+def _pad_time_series(X, n_delays, fill_mean=True):
```

Comment:
don't we have some function like this already implemented to do the zero padding for things like FFT? (e.g., zero padding up to a length of 2**N). Just asking for the sake of DRY :)